src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs

```diff
@@ -67,9 +67,30 @@ public static EmissionsForecastDTO FromEmissionsForecast(EmissionsForecast emiss
             DataStartAt = emissionsForecast.DataStartAt,
             DataEndAt = emissionsForecast.DataEndAt,
             WindowSize = (int)emissionsForecast.WindowSize.TotalMinutes,
-            OptimalDataPoint = EmissionsDataDTO.FromEmissionsData(emissionsForecast.OptimalDataPoint),
+            OptimalDataPoints = GetOptimalDataPointsDTO(emissionsForecast.OptimalDataPoints),
```

Comment:
_If changing the contract:_

Consider doing this the same way as the line below it.

```suggestion
            OptimalDataPoints = emissionsForecast.OptimalDataPoints.Select(d => EmissionsDataDTO.FromEmissionsData(d))!,
```

Reply:
@dylHall - do you think youc an make this change?  it would be nice to have but possibly to do as a quick update (due to the need to get the contract updates through to dev)